chain-tx-enclave/tx-validation/app/README.md

```diff
@@ -0,0 +1 @@
+tx-validation-app was moved to chain-abci
```

Comment:
Why not just remove the directory?

Reply:
There are a lot of "tx-validation-app" references around in documentation, old deployment scripts etc. -- so it's more of a note for the outside world (and @leejw51crypto when he forgets about it :)) that I thought would be good to put in temporarily.